docs/reference/faq-troubleshooting.md

```diff
@@ -1,17 +1,75 @@
 # FAQ & Troubleshooting
 
-## What can I do about `error parsing torrent at http://…`?
+### What can I do about `error parsing torrent at http://…`?
 
 This means that the jacket download link didn't resolve to a torrent file. It's
 possible you got rate-limited so you might want to try again in a day or more.
 Otherwise, just ignore it. There's nothing cross-seed will be able to do to fix
 it.
 
-## rtorrent injected torrents don't check (or start at all) until force rechecked
+### rtorrent injected torrents don't check (or start at all) until force rechecked
+
 Remove any `stop_untied=` schedules from your .rtorrent.rc.
 
 Commonly: `schedule2 = untied_directory, 5, 5, (cat,"stop_untied=",(cfg.watch),"*.torrent")`
 
-## Failed to inject, saving instead.
+### Failed to inject, saving instead.
+
+Best way to start troubleshooting this is to check the `verbose.\*.log` and find this specific event.
+
+You will be able to see the circumstances around the failure, and start investigating why this occurred.
+
+### My data-based results are paused at 100% after injection
+
+This is by design and due to the way data-based searches function. This is done to prevent downloading missing files
+or files that failed rechecking automatically. Until there is a way to guarentee you won't end up downloading
+extra (not cross-seeding) this is the best solution.
+
+Consider using infoHash if you are racing using data-based searches to prevent mismatches.
+
+### My data-based search is searching torrent files!
+
+Torrent files in torrentDir take precedence over data files with the same name and are deduplicated. If you need to search
+specifically the dataDir, use an empty directory for torrentDir temporarily. This will strictly search the data.
+
+### Error: ENOENT: no such file or directory
+
+This is usually caused by broken symlinks in your dataDir during a data-based search. Check the `/logs/verbose.\*.log` files for the file causing this.
+
+### What linkType should I use? (data-based searching)
+
+Your options are `hardlink` or `symlink`. These operate in seperate ways, and depending on your workflow you should choose appropriately. This is a brief description, however
+a more in depth guide is available at [Trash's Hardlinking Guide](https://trash-guides.info/Hardlinks/Hardlinks-and-Instant-Moves/).
+
+- symlinks are a "shortcut" of a sort, pointing at the original file from a new location. This can be used across mounts (docker) or partitions/drives. The issue is that if the original file is deleted when you remove a torrent, the torrents in your client using the symlink will "break" and you will receive errors. If this sounds like a hassle, consider reading further about hardlinks.
+
```

Comment:
also discuss symlink advantages (broken symlinks take no space) and disadvantages of hardlinks (broken links still take up space)

also plug qbm for hardlinks I think, since that really is the optimal setup imo

Reply:
That's a benefit for hardlinks for me :)

Covering it in next commit.